consensus/blake3pow/consensus.go

```diff
@@ -777,7 +780,7 @@ func (blake3pow *Blake3pow) FinalizeAndAssemble(chain consensus.ChainHeaderReade
 	nodeCtx := blake3pow.config.NodeLocation.Context()
 	if nodeCtx == common.ZONE_CTX && chain.ProcessingState() {
 		// Finalize block
-		if _, _, err := blake3pow.Finalize(chain, nil, header, state, true, parentUtxoSetSize, utxosCreate, utxosDelete); err != nil {
+		if _, _, err := blake3pow.Finalize(chain, nil, header, state, true, parentUtxoSetSize, utxosCreate, utxosDelete, nil); err != nil {
```

Comment:
if you supply nil here, it doesnt look like you update the supplyRemovedQi variable, so shouldnt it be sent as zero?

Reply:
Only the worker sends it as nil, it's not needed in the worker